pkg/3scale/amp/operator/memcached_options_provider.go

```diff
@@ -75,6 +75,12 @@ func (m *MemcachedOptionsProvider) podTemplateLabels() map[string]string {
 		labels[k] = v
 	}
 
+	if m.apimanager.Spec.System.MemcachedLabels != nil {
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)